src/Storages/ObjectStorage/DataLakes/Iceberg/IcebergMetadata.cpp

```diff
@@ -1,4 +1,5 @@
 #include "Disks/ObjectStorages/StoredObject.h"
+#include "Storages/ObjectStorage/DataLakes/Iceberg/IcebergMetadataFilesCache.h"
```

Comment:
it would be nice if we arrange header files better

Reply:
You can upgrade to clang 20 and the problem will gone (https://github.com/llvm/llvm-project/pull/67749)